### PR TITLE
added the stylings for the web version of the page

### PR DESCRIPTION
**Pull request by Trust-007**
Added media query to css for the web version of the page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -381,3 +399,85 @@ form textarea {
...
+
+@media (min-width: 768px) {
+  nav .menu-bar {
+    display: none;
+  }
+
+  nav {
+    justify-content: space-around;
+    gap: 240px;
+  }
```
Comment (mckent05) [inline index.css:411]: - Nice work you have done so far 👍 , but there are still some small differences between the Figma template and your design 😄. the width of your navbar doesn't exactly match the design. Please try to fix this. Know that replicating the exact specifications gives confidence to recruiters in your ability to design the exact webpage they'll require 🙏 

- Please also note that the background images does not exactly match what is in the figma template, please try to fix that 🙏 
Your project looks like this:
- Please pay attention to the margin difference between your logo and the left border of the site and also the margin difference between the nav links and the right border of the site.
figma 1 desktop project

Figma template:
figma 1 desktop figma

File: index.css [hunk 2]
```diff
@@ -381,3 +399,85 @@ form textarea {
...
+  .project-item:nth-of-type(3) {
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .project-item:nth-of-type(2),
+  .project-item:nth-of-type(4) {
+    flex-direction: row-reverse;
+    gap: 20px;
+  }
```
Comment (mckent05) [inline index.css:447]: - Nice work you have done so far 👍 , but there are still some small differences between the Figma template and your design 😄. the width of your project cards doesn't exactly match the design. Please try to fix this. Know that replicating the exact specifications gives confidence to recruiters in your ability to design the exact webpage they'll require 🙏

Your project looks like this:

   - Please pay attention to the width of each of your project cards 🙏. The width in the Figma design takes a little bit more space than you have implemented in your design. I suggest you look into fixing this 👍 
 work card figma

Figma template-
work card student

File: index.css [hunk 3]
```diff
@@ -381,3 +399,85 @@ form textarea {
...
+
+  .about-myself-skills {
+    width: 100%;
+    margin-right: 5%;
+  }
+
+  .contact-description {
+    margin-inline: 32%;
+  }
+
```
Comment (mckent05) [inline index.css:475]: - Please also note that the background images in your contact section does not exactly match what is in the figma template, please try to fix that 🙏

Your project looks like this:
contact figma 1 temlate

Figma template-
figma desktop contact

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Trust-007, 

# STATUS: CHANGES REQUIRED ♻️ ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Linters are passing ✔️ ✔️ 
- PR title and summary are descriptive 💯 💯 
- Your portfolios gradually coming together and starting to look complete 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @Trust-007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Github flow ✔️ 
- [ ] Good commit messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**